loader.cfc

```diff
@@ -73,8 +73,11 @@ component accessors=true {
 	**/
 	public component function load(
 		required component cfc,
-		required struct data = {}
+		required any data = {}
 	) {
+		if ( !IsStruct(arguments.data) ) {
+			arguments.data = DeserializeJson(arguments.data);
+		}

```

Comment:
This is the actual code update. Nice and simple. Doesn't really seem to make the benchmarks any slower.